app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -59,7 +59,7 @@ def show
     param :force, :bool, :desc => N_("force content view promotion and bypass lifecycle environment restriction")
     param :environment_id, :identifier
     def promote
-      is_force = params[:force].is_a?(String) ? params[:force].to_bool : params[:force]
+      is_force = Cast.to_bool(params[:force])
```

Comment:
Not a huge fan of the usage (would prefer value.to_bool) but I'm unsure of another way to do that without monkeypatching.